pkg/daemon/cleanup/cleanup.go

```diff
@@ -77,6 +77,7 @@ func Deconfigure(cfg *rest.Config, ctx context.Context, preserveNode bool) error
 	if err != nil {
 		klog.Exitf("could not create service manager: %s", err.Error())
 	}
+	defer svcMgr.Disconnect()
```

Comment:
I assume you ran into an issue because handles were not being closed. Please update the commit message with the issue you ran into.